app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,16 @@
+require 'JSON'
+class Api::V1::ReviewsController < ApiController
+  def create
+    location = Location.find(params["location_id"])
+    new_review = Review.new(review_params)
+    new_review.user = current_user
+    new_review.location = location
+    new_review.save
+    render json: new_review
+  end
```

Comment:
this is grat that you have this working, but: 
- there should be an `if...else` condition for if the review does not save, that includes sending error messages back 
- you shouldnt need to rquire `JSON`